### PR TITLE
add kleingarten

### PR DESCRIPTION
**Pull request by freifunkoperator**

### CODE REVIEW

**Comment by pktpls** [comment]
But I'm cherry-picking that unifiac-mesh 2ghz fix to main

File: locations/hdk-15.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+  # MGMT
+  - vid: 42
+    role: mgmt
+    prefix: 10.248.5.64/28
+    gateway: 1
+    dns: 1
+    ipv6_subprefix: 1
+    assignments:
+      # 10.248.5.1/32
```
[the remark this answers is not in the document]
Comment (Noki) [reply locations/hdk-15.yml]: changed

**Comment by Noki** [comment]
Fixed the mentioned issues and linter issues. In my opinion this is good to go and flashing new firmware to hdk-17 will resolve some issues the device had with meshing in the past.

File: locations/hdk-30.yml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+  # MESH - 2.4 GHz 802.11s
+  - vid: 21
+    role: mesh
+    name: mesh_2ghz
+    prefix: 10.248.4.209/32
+    ipv6_subprefix: -2
+    # make mesh_metric(s) for 2GHz worse than 5GHz
+    mesh_metric: 1024
+    mesh_metric_lqm: ['default 0.8']
```
[the remark this answers is not in the document]
Comment (Noki) [reply locations/hdk-30.yml]: removed

**Pull request merged**